src/api/contributionsApi.ts

```diff
@@ -14,9 +15,34 @@ const EPIC_TEST_LIST_FILE_LOOKUP: EpicTestListTestFileLookup = {
     LIVEBLOG: 'liveblog-epic-tests.json',
 };
 
-export const fetchConfiguredEpicTests = async (testList: EpicTestList): Promise<EpicTests> => {
+export const variantHasArticleCountCopy = (variant: EpicVariant): boolean => {
+    const { paragraphs, heading, highlightedText } = variant;
+    return (
+        (!!heading && containsArticleCountPlaceholder(heading)) ||
+        paragraphs.some(containsArticleCountPlaceholder) ||
+        (!!highlightedText && containsArticleCountPlaceholder(highlightedText))
+    );
+};
+
+export const fetchConfiguredEpicTests = async (testList: EpicTestList): Promise<EpicTest[]> => {
     const file = EPIC_TEST_LIST_FILE_LOOKUP[testList];
     const key = `epic/${isProd ? 'PROD' : 'CODE'}/${file}`;
 
-    return fetchS3Data('gu-contributions-public', key).then(JSON.parse);
+    return fetchS3Data('gu-contributions-public', key)
+        .then(JSON.parse)
+        .then(json => {
+            const { tests } = json;
+            if (Array.isArray(tests)) {
```

Comment:
Is this because `tests` might be `undefined`? Or is it more for keeping ts happy?

Reply:
yes (though hopefully TS will complain as well!). This is the raw json from S3, it's good to do some validation, though there's lots more that we _could_ do

Reply:
sorry just re-read the question - `tests` of course shouldn't ever be `undefined` in the json!

Reply:
yeah makes sense. Could always look into using `zod` or something like that in the future!